docs/core/tools/dotnet-sln.md

```diff
@@ -87,66 +108,50 @@ Removes a project or multiple projects from the solution file.
 #### Synopsis
 
 ```dotnetcli
-dotnet sln [<SOLUTION_FILE>] remove <PROJECT_PATH>
+dotnet sln [<SOLUTION_FILE>] remove <PROJECT_PATH> [<PROJECT_PATH>...]
 dotnet sln [<SOLUTION_FILE>] remove [-h|--help]
 ```
 
 #### Arguments
 
 - **`SOLUTION_FILE`**
 
-  The solution file to use. If not specified, the command searches the current directory for one. If there are multiple solution files in the directory, one must be specified.
+  The solution file to use. The command searches the current directory for one if it's left unspecified, failing if there are multiple solution files.
 
 - **`PROJECT_PATH`**
 
-  The path to the project to remove from the solution. Remove multiple projects by adding one after the other separated by spaces. Unix/Linux shell [globbing pattern](https://en.wikipedia.org/wiki/Glob_(programming)) expansions are processed correctly by the `dotnet sln` command.
+  The path to the project or projects to add to the solution. Unix/Linux shell [globbing pattern](https://en.wikipedia.org/wiki/Glob_(programming)) expansions are processed correctly by the `dotnet sln` command.
 
 #### Options
 
 - **`-h|--help`**
 
-  Prints out a short help for the command.
-
-### `list`
+  Prints out configurable options for the command.
```

Comment:
This looks different from the previous description for help.

Reply:
I included two versions to see which you like better, but I don't like the current one. It's bad style to include a word in its definition, and this usage for "help" is awkward in English in my opinion, so I would prefer to use one of my versions. Do you like either?

Reply:
Would we change in the CLI too? It's not exactly the same but it says help too.

I don't think that --help is only about options so I think both changes are a bit misleading.

Reply:
I would.

That's true. How about one of:
"Emits a description of how to use dotnet sln list"
"Prints out a description of how to use dotnet sln list"
"Describes how to use dotnet sln list"
possibly replacing "dotnet sln list" with "this command"?

Reply:
I vote for "Prints out a description of how to use the command."  Whatever we end up with here we can propagate to the other CLI docs later.